CONTRIBUTING.md

```diff
@@ -0,0 +1,181 @@
+# Contributing to maap-py
+
+Thank you for contributing to maap-py!
+
+### Table of Contents
+
+- [How to Get Help](CONTRIBUTING.md#how-to-get-help)
+- [How to Contribute](CONTRIBUTING.md#how-to-contribute)
+    - [Reporting Bugs](CONTRIBUTING.md#reporting-bugs)
+    - [Feature Requests](CONTRIBUTING.md#featureenhancement-requests)
+    - [Pull Requests](CONTRIBUTING.md#pull-requests)
+    - [Branches](CONTRIBUTING.md#branches)
+    - [Changelog](CONTRIBUTING.md#changelog)
+    - [Testing](CONTRIBUTING.md#testing)
+    - [Reviewing](CONTRIBUTING.md#reviewing)
+- [Style Guides](CONTRIBUTING.md#style-guides)
+    - [Python Style Guide](CONTRIBUTING.md#python-style-guide)
+    - [Documentation Style Guide](CONTRIBUTING.md#documentation-1)
+
+## How to Get Help
+
+The easiest way to get help or ask a question regarding maap-py is filing an [issue](https://github.com/MAAP-Project/maap-py/issues).
+
+## How to Contribute
+
+### Reporting Bugs
+
+Bugs can be reported by creating a new Github issue in this repository. 
+Please use the `bug` label on any bug issue created. Provide detailed 
+instructions on how to reproduce the bug if possible. Include the version 
+of maap-py that produced this bug and specific error messages.
+
+### Feature/Enhancement Requests
+
+Feature/enhancement requests are tracked with 
+Github issues. Please use 
+an appropriate label:
+
+- `documentation`
+    - Docstring or sphinx documentation changes
+- `enhancement`
+    - Improvement of an existing feature or description of a new feature
+- `bug`
+    - An error with existing functionality
+    
+Feature and enhancement requests should contain a detailed description 
+of the desired behavior and rationale.
+
+### Pull Requests
+
+Please ensure all pull requests follow the PR 
+[template](/.github/pull_request_template.md). This is to ensure
+
+* A reviewer understands what is being changed
+* A reviewer understands why it is being changed
+* A reviewer understands how the changes have been verified
+* A reviewer is confident the changes work as expected and won't break existing functionality.
+
+The name of the pull request should be meaningful, and if the pull 
+request is addressing a Github issue, should match the name of the 
+Github issue. 
+
+If any performance improvements are being made, include graphs and charts.
+
+### Branches
+
+- `develop` (protected)
+    - Code actively being developed
+- `master` (protected)
```

Comment:
```suggestion
- `main` (protected)
```

Reply:
The branch is currently `master`. We'd have to change this when we rename master -> main. Do you think we should do that now?

Reply:
Sorry, let's leave it to master. And tackle the change to main in another ticket.